utils/genprimopcode/genprimopcode.cabal

```diff
@@ -31,5 +31,7 @@ Executable genprimopcode
                    AccessOps
     Build-Depends: base       >= 4   && < 5,
                    array
+    default-extensions:
+      UnboxedSums
```

Comment:
Do we still need this?

Reply:
I think it was added because Happy generates code that uses UnboxedSums but doesn't generate the pragmas to enable them...

I couldn't find an upstream issue but it would be similar to https://github.com/haskell/happy/issues/183

We should probably keep it to be on the safe side.